.github/workflows/testBuildHybrid.yml

```diff
@@ -5,52 +5,45 @@ on:
     inputs:
       PULL_REQUEST_NUMBER:
         description: Pull Request number for correct placement of apps
-        required: true
-  pull_request_target:
-    types: [opened, synchronize, labeled]
-    branches: ['*ci-test/**']
-
-env:
-  PULL_REQUEST_NUMBER: ${{ github.event.number || github.event.inputs.PULL_REQUEST_NUMBER }}
+        required: false
+        default: ''
+      OLD_DOT_PULL_REQUEST_NUMBER:
+        description: Old Dot Pull request number to build the app from. It will take precedence over MOBILE-EXPENSIFY from ND PR if both specified. 
```

Comment:
```suggestion
        description: Old Dot Pull request number to build the app from. It will take precedence over MOBILE-EXPENSIFY from ND PR description if both specified. 
```
?

Reply:
Thanks!